runners/google-cloud-dataflow-java/worker/build.gradle

```diff
@@ -137,6 +137,14 @@ applyJavaNature(
             relocate("org.eclipse.jetty", getWorkerRelocatedPath("org.eclipse.jetty"))
             relocate("javax.servlet", getWorkerRelocatedPath("javax.servlet"))
 
+            // Use Caffeine cache instead of Guava cache.
+            // Context: https://guava.dev/releases/snapshot/api/docs/com/google/common/cache/CacheBuilder
+            // For Java8+ and less than Java11, use versions 2.x.x per https://github.com/ben-manes/caffeine.
```

Comment:
We probably want the version to be conditional on the java version being used then.

@Abacn Could you provide some expertise here? Does this require vendoring if it's just used within the worker? Any other gotchas about adding a dependency?